wcivf/apps/elections/templates/elections/includes/_city_of_london_registration_details.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+
+<li>
+    <details open>
+        <summary>
+            <h2>
+                <span aria-hidden="true"></span>
+                {% blocktrans trimmed %}Register to vote{% endblocktrans %}
+            </h2>
+        </summary>
+        <p>
+            {% blocktrans trimmed %}
+                City of London council elections do not use the same electoral register as other elections.
+            {% endblocktrans %}
+        </p>
+        <p>
+            {% blocktrans trimmed %}
+                Both residents and city workers are eligible to vote. There is one register
+                published annually, and the deadline to apply is 30 November each year.
+                The new register comes into force on 16 February each year,
+                and cannot be modified after that date. For more information, visit the
+                <a href="https://www.speakforthecity.com/">Speak for the City</a>
+                website or contact City of London electoral services.
```

Comment:
This was Peter's suggested text, but we can tweak it if you want now we have the logic in place to show/hide text in the right situations.

I think the rules are sufficiently simple here that we might as well just hard-code the dates like this.